src/popup/components/tweeting.tsx

```diff
@@ -1,5 +1,15 @@
 import * as React from 'react'
 
 export function Tweeting({ at }: { at: string }): JSX.Element {
-  return <p className="tweet-in-progress">Tweeting your feedback to {at}</p>
+  return (
+    <>
```

Comment:
Let's make these `div`s with `className`s just to make it a bit easier for debugging and in case we want to style them directly

```typescript
<div className="tweeting">
  <div className="tweeting-content">
    ...
```